build.gradle

```diff
@@ -562,6 +562,7 @@ project(':clients') {
     compile libs.lz4
     compile libs.snappy
     compile libs.slf4jApi
+    compile libs.jacksonDatabind
```

Comment:
Sorry, I wasn't necessarily suggesting a specific grouping yet, just identifying the problem and a solution. I think we'd need to propose a way to break them down to the mailing list. Anything we add to `kafka-clients` is basically stuck there permanently (or moved to a new dependency of `kafka-clients`).